cmake/llvm-version.txt

```diff
@@ -1 +1 @@
-179e174945b6c0da462c534504720c9544aebf84
+5adcfd4c17826b2b8f023881baa1c7f79cb23920
```

Comment:
It should be fine to combine the llvm version update and other changes in a single PR at this stage, but it'd be better to split this in future?

Reply:
It is to trigger LLVM rebuilt in CI. We need a new flag in cmake options of LLVM.